src/Robo/Plugin/Commands/ValidateConfigCommands.php

```diff
@@ -35,28 +45,45 @@ public function __construct()
     /**
      * Validate Drupal configuration status.
      *
-     * @param string $siteDir
-     *   The Drupal site directory.
-     *
+     * @param string $siteDirs
+     *   A comma separated list of Drupal site directories.
```

Comment:
Did we do this vs accepting an array for backwards compatibility?

Or did we do this since that makes it easier to pass the value from the CLI?

Reply:
CLI. This is a publicly-exposed command usually called from the command-line, so I wanted to keep that possible.

Reply:
I figured that was the idea half-way through my comment.